### PR TITLE
People might want to make it clear that a64 is there too

### PR DESCRIPTION
**Pull request by peaktogoo**
As topic said

### CODE REVIEW

File: build-dakkar.sh [hunk 1]
```diff
@@ -54,6 +54,7 @@ Variants are dash-joined combinations of (in order):
 * processor type
   * "arm" for ARM 32 bit
   * "arm64" for ARM 64 bit
+  * "a64" for ARM 32 bit system with 64 bit binddr
```
Comment (phhusson) [inline build-dakkar.sh]: typo, binder not binddr

**Comment by phhusson** [comment]
good for me except for the typo

File: build-dakkar.sh (hunk 1 above)
Comment (peaktogoo) [reply build-dakkar.sh]: my bad ;&;

**Comment by peaktogoo** [comment]
Typo fixed

**Comment by peaktogoo** [comment]
Also fixed the pixel90 repository link

**Comment by phhusson** [comment]
Oops sorry, I rebased your commit to have a proper history, it lost your pixel90 commit in the battle

**Comment by peaktogoo** [comment]
> Oops sorry, I rebased your commit to have a proper history, it lost your pixel90 commit in the battle

Rebase again? I'm kjnda noob at git thing.

**Comment by phhusson** [comment]
git pull --rebase https://github.com/peaktogoo/treble_experimentations.git master
should do the trick

**Pull request merged**